ts/features/fci/components/LinkedText.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import * as O from "fp-ts/lib/Option";
+import { H4 } from "../../../components/core/typography/H4";
+import { Link } from "../../../components/core/typography/Link";
+import { WithTestID } from "../../../types/WithTestID";
+
+type Props = WithTestID<{
+  text: string;
+  url: string;
```

Comment:
It's not clear for an external user of the component what this "url" means.
What about an object with **tag** as key and **replacement** as value?

Reply:
I'll change the "url" name to a clearer one for now. There will be a refactor dependent on this coming soon related to the https://pagopa.atlassian.net/browse/SFEQS-1230